fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateRoutineLoadStmt.java

```diff
@@ -485,6 +511,11 @@ public void checkJobProperties() throws UserException {
         } else {
```

Comment:
Are the format and parameter the same with the CSV stream load?

Reply:
> Are the format and parameter the same with the CSV stream load?

Good comment. I will change the name of trimspace to trim_space, just like stream load.